index.html

```diff
@@ -644,6 +678,102 @@ <h3>
             </ol>
           </li>
         </ol>
+        <p>
+          To <dfn>update touches</dfn> for |gamepad:Gamepad|, run the following
+          steps:
+        </p>
+        <ol class="algorithm">
+          <li>Let |surfaceId:unsigned long| be 0.
+          </li>
+          <li>Remove any existing events from
+          {{Gamepad}}.{{Gamepad/[[touches]]}}.
+          </li>
+          <li>Repeat the following steps for each touch surface on |gamepad| in
+          touch surface enumeration order:
+            <ol>
+              <li>Let |surfaceDimensions| be an `null`.
+              </li>
+              <li>If the touch surface exposes maximum surface dimensions in
+              device units:
+                <ol>
+                  <li>Set |touchEvent|.{{GamepadTouch/surfaceDimensions}}[0] to
```

Comment:
Note, move away from calling it "*Event". Call it |gameTouch|, and 
```suggestion
                  <li>Let |gamepadTouch| be a newly created {{GamepadTouch}.</li>
                  <li>Set |touchEvent|.{{GamepadTouch/surfaceDimensions}}[0] to
```